### ISSUE TITLE
Imagescrollers are not accessible

### ISSUE THREAD

**Issue by cmb69** [cmb69/imagescroller_xh#31]
As is, I'm afraid they are not accessible with keyboard navigation, badly accessible on touch-screens (should support swiping), and super bad wrt assistive technologies such as screen readers.

https://www.w3.org/WAI/tutorials/carousels/ might be helpful.
